sourcecode/hub/resources/views/components/preview-modal.blade.php

```diff
@@ -0,0 +1,19 @@
+<!-- Preview Modal -->
+<div class="modal fade" id="previewModal" data-bs-backdrop="static" data-bs-keyboard="false" tabindex="-1" aria-labelledby="previewModalLabel" aria-hidden="true">
+    <div class="modal-dialog modal-dialog-centered modal-lg" style="max-height: 80vh;">
```

Comment:
Avoid using style, use class. The class should have a generic name so that it can be reused, look at the naming used by Bootstrap (https://getbootstrap.com/docs/5.3/utilities/sizing/#relative-to-the-viewport)